packages/html/stories/Events.stories.ts

```
@@ -0,0 +1,177 @@
+  );
+
+  // Enables tooltips, new connections and panning
+  graph.setPanning(true);
+  graph.setTooltips(true);
+  graph.setConnectable(true);
+
+  // Automatically handle parallel edges
+  const layout = new ParallelEdgeLayout(graph);
+  const layoutMgr = new LayoutManager(graph);
+  layoutMgr.getLayout = function (cell, _eventName: string): GraphLayout | null {
+    if (cell?.getChildCount() ?? 0 > 0) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)